src/shared/utils/exportData/prepareData.ts

```diff
@@ -41,7 +42,16 @@ export const prepareData = (
           }),
         );
       }, [] as ReturnType<typeof getReportCSVObject>[]);
-      const reportData = acc.reportData.concat(...answers);
+
+      const subscaleSetting = data.decryptedAnswers?.[0]?.subscaleSetting;
```

Comment:
You added subscales calc only for first answer. Can you confirm it should be skipped for others?

Reply:
Yes